core/src/main/java/org/apache/iceberg/util/ArrayUtil.java

```diff
@@ -92,6 +94,8 @@ private ArrayUtil() {
   }
 
   /**
+   * <p>The code from `org.apache.commons:commons-lang3`
```

Comment:
Nit: I don't think you need an initial `<p>` tag.

What I've seen elsewhere in the code base is that the first line is written without `<p>`, and then subsequent paragraphs have a `<p>` tag between them where the `<p>` tag is on its own line. We tend to not use closing `</p>` tags as the java doc specification doesn't require them.

So here we would have
```java
/**
 * This code is borrowed from `org.apache.commons:commons-lang3`.
 * <p>
 * Converts an array of object Bytes to primitives.
 * <p>
 */
```

Reply:
make sense. updated